style-editor.css

```diff
@@ -164,6 +164,16 @@ p:not( [class*='-font-size'] ) {
 	border: 10px solid var( --color-accent-border );
 }
 
+.is-style-emphasis {
+	color: var( --color-accent-text );
+	font-family: var( --font-header );
+	font-weight: bold;
+	letter-spacing: -0.0415625em;
+}
+.is-style-emphasis:not( [class*='-font-size'] ) {
+	font-size: 1.95em;
```

Comment:
What's the reason for this font-size being smaller in the editor?

Reply:
Copied from the parent theme to be consistent. I agree we should look into having consistency, but let's track in another issue?